scripts/generate-openapi.ts

```
@@ -70,7 +69,17 @@ async function main(): Promise<void> {
   } finally {
-    await app.close();
+    const cleanupResults = await Promise.allSettled([
+      app.close(),
+      import('../src/db/client.js').then(({ db }) => db.end()),
+      import('../src/db/redis.js').then(({ redis }) => redis.quit()),
+    ]);
+    const cleanupFailures = cleanupResults
+      .filter((result): result is PromiseRejectedResult => result.status === 'rejected')
+      .map((result) => result.reason);
+    if (cleanupFailures.length > 0) {
+      throw new AggregateError(cleanupFailures, 'OpenAPI generator cleanup failed');
```

Biome (2.5.3)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)